module/scripts/index/commons-source-ui.js

```diff
@@ -2,7 +2,37 @@
 Refine.CommonsSourceUI = function(controller) {
     this._controller = controller;
 };
-var categoryJsonObj = [];
+
+Refine.CommonsSourceUI.prototype.addRow = function() {
+
+  var tr = $(`<tr id="catRow">
+  <td class="row-index text-center">
+  <input size="72"  width="70%" class="category-input-box"/></td>
```

Comment:
Rather than adding a `width` and `style` attributes to all those elements, it would be cleaner to define those attributes in CSS. You already have class names on most DOM elements so it should not be too hard to migrate to that.
It looks like you have not got any stylesheet in this extension yet, so you will need to introduce that. Like Javascript files, those need to be registered in your `controller.js` file. You can check how it is done in the Wikidata extension for instance: https://github.com/OpenRefine/OpenRefine/blob/master/extensions/wikidata/module/MOD-INF/controller.js

Reply:
Ah sorry I did not see that you already had a style file indeed! Your additional styling can go there, then.

Reply:
we have
  `//Style files to inject into /index page
  ClientSideResourceManager.addPaths(
    "index/styles",
    module,
    [
      "styles/commons-importing-controller.less",
      "externals/suggest/css/suggest-4_3.min.css"
    ]
  );`
are these not the right ones?
Should I define the width and style attributes in a new .css file?
or add them in that existing commons-importing-controller.less one?